integration-tests/utils/integration-utils.js

```diff
@@ -191,10 +191,30 @@ const dismissNotifications = async page => {
   return !!notificationCloseButtons.length && delay(1000) // delayed for alerts to animate off
 }
 
+const loadSignInPageAndWaitForReady = (page, url, timeout = 30 * 1000) => {
+  return Promise.all([
+    page.goto(url, waitUntilLoadedOrTimeout(timeout)),
+    Promise.race([
```

Comment:
Should `page.goto` and the two `waitForXPath` in `Promise.race` start at the same time? Or should the `waitForXPath`s be run after `page.goto` is done?

Reply:
👍🏻 It should be in sequential order, `goto` first and done, then `waitForXpath` in `Promise.race`. It's fixed now.